.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'Closing issue due to lack of activity!'
```

Comment:
Could we add `days-before-stale` and `days-before-close` as specific times, instead of relying on the hidden defaults of `60` and `7` by default?